src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Mention.java

```diff
@@ -36,6 +38,21 @@ public Mention(long id, String idStr, String text, Date createdAt, String fromUs
         this.queryId = queryId;
     }
 
+    private static Mention tweetToMention(Tweet tweet, long queryId){
+        return new Mention(tweet.getId(), tweet.getIdStr(),tweet.getText(), tweet.getCreatedAt(), tweet.getFromUser(),
+                tweet.getProfileImageUrl(), tweet.getToUserId(), tweet.getFromUserId(), tweet.getLanguageCode(),
+                tweet.getSource(), queryId);
+    }
+
+    public static List<Mention> tweetsToMentions(List<Tweet> tweets, long queryId) {
+        List<Mention> mentions = new ArrayList<>();
```

Comment:
You can change this method to 

```java
return tweets.stream
    .map(tweet -> tweetToMention(tweet, queryId))
    .collect(Collectors.toList());
```

Reply:
Nice, that is the map method.
I should have thought of it since it is in Python and Haskell. Still I need to read more about it.